hwangJeongSoo/app.js

```diff
@@ -0,0 +1,91 @@
+const http = require("http");
+
+require("dotenv").config()
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const { DataSource } = require('typeorm')
+
+const appDataSource = new DataSource({
+    type : process.env.TYPEORM_CONNECTION,
+    host : process.env.TYPEORM_HOST,
+    port : process.env.TYPEORM_PORT,
+    username : process.env.TYPEORM_USERNAME,
+    password : process.env.TYPEORM_PASSWORD,
+    database : process.env.TYPEORM_DATABASE 
+})
+
+appDataSource.initialize() 
+    .then(()=>{
+        console.log("Data Source has been initialized")
+    })
+
+const app = express();
+
+app.use(express.json())
+app.use(cors())
+app.use(morgan('dev'))
+
+app.get("/ping", (req, res) =>{
+	res.status(201).json({ message : "pong"});
+    });
+
+
+//2. signup
+app.post("/signup", async(req, res, next)=>{
+    const { name, email, profile_image, password } = req.body
+
+
+await myDataSource.query(
+    `INSERT INTO users(
+        name,
+        email,
+        profile_image,
+        password
+    ) VALUES (?, ?, ?, ?);
+    `,
+    [ name, email, profile_image, password ]
+); 
+     res.status(201).json({ message : "successfully created"});
+    })
```

Comment:
코드의 가독성을 높이는 방법에는 여러가지가 있습니다.

그 중에서 들여쓰기 (indentation)는 해당 코드가 어떤 함수에 포함되는지 확인할 수 있을 것입니다.

마이너한 부분이지만 가독성이 좋은 코드로 수정해주세요~!
```suggestion
//2. signup
app.post("/signup", async(req, res, next)=>{
    const { name, email, profile_image, password } = req.body

    await myDataSource.query(
        `INSERT INTO users(
            name,
            email,
            profile_image,
            password
        ) VALUES (?, ?, ?, ?);
        `,
        [ name, email, profile_image, password ]
    ); 
     res.status(201).json({ message : "successfully created"});
})
```